src/frontend/screens/DownloadManager/components/ProgressHeader/index.tsx

```diff
@@ -50,6 +51,11 @@ export default function ProgressHeader(props: {
   const showDownloadBar =
     props.state !== 'idle' && props.appName && progress.percent
 
+  const progressBytes =
+    typeof progress.bytes === 'number'
+      ? size(progress.bytes * 1024 * 1024)
+      : progress.bytes
+
```

Comment:
this seems off, if progress.bytes is typed as a string why is it possible it is a number?

Reply:
yeah, so, its complicated.
For Legendary and GOGDL it is a string, but for hyperplay its is a number.
We need to refactor those in the future to be the same for all.